src/Debugger.cpp

```diff
@@ -2873,7 +2870,18 @@ void Debugger::set_initial_debugger_state() {
 
 	const QString filename = session_filename();
 	if(!filename.isEmpty()) {
-		load_session(filename);
+		SessionError session_error;
+		if(!SessionManager::instance().load_session(filename, session_error)) {
+			QMessageBox::warning(
+				this,
+				tr("Error Loading Session"),
+				tr(session_error.getErrorMessage().toUtf8().constData())
```

Comment:
1. Why the `.toUtf8().constData()`? `QObject:tr` already takes a `QString` as input, these conversions are superfluous, or am I missing something?
2. Qt Linguist won't find the string to give to translators, so this call to `tr` will never work (but see `QT_TR_NOOP`).

Reply:
Sorry, this wasn't my intention

Reply:
This was just a workaround to shut the compiler up. I was planing on using QT_TR_NOOP but had some trouble finding out if this would be the best way to got.

Reply:
Ah, sorry, point 1) is indeed wrong, there's no such overload.